frontend/src/assets/css/typography.scss

```diff
@@ -0,0 +1,22 @@
+h3 {
+  font-size: 58px;
```

Comment:
Do we have a default value for font-size? 
I think it is better to use **rem**. Because in this case if we need to update font-size we will need to update all px values

Reply:
@nikita-remeslov @oleksandr-danylchenko liner doesn't allow me to use `rem`: `Unexpected unit "rem"  unit-disallowed-list`